setup/github-bot/ansible-playbook.yaml

```diff
@@ -0,0 +1,64 @@
+---
+- hosts: github-bot
+  remote_user: root
+
+  tasks:
+
+    - include_vars: ansible-vars.yaml
+      tags: vars
+
+    - name: General | Install baseline packages
+      apt: name="{{ item }}" state=latest
+      with_items: "{{ baseline_packages }}"
+      tags: general
+
```

Comment:
I would bootstrap similarly to how we do it in other packages -- sometimes python isn't available and I reckon this would break if you don't have aptitude installed.

Check the fedora example I gave where we set `get_facts` to `False` and run `raw` commands to install the baseline stuff.